tests/tests/test_patterns.py

```diff
@@ -22,16 +22,26 @@ def log_repr(x):
 
 VALUES = [
     object(),
-    True, False,
-    0, -1, -1.0, 1.23,
-    b'abc', b'abcd',
-    u'abc', u'abcd',
-    (), (1, 2, 3),
-    [], [1, 2, 3],
-    {}, {'a': 1, 'b': 2},
+    True,
+    False,
+    0,
+    -1,
+    -1.0,
+    1.23,
+    b"abc",
+    b"abcd",
+    "abc",
```

Comment:
```suggestion
    u"abc",
```

Reply:
These are all from Black - since we use `unicode_literals`, the prefix is redundant, and apparently this means that it gets removed.